python/perf-kernels/README.md

```diff
@@ -42,6 +42,7 @@ This script contains the Flash Attention kernel with the following support
 - Multi and Grouped Query attention
 - ALiBi bias
 - Matrix bias
+- Int8 quantization
```

Comment:
Can you add a bit more details under this? What happens if only kv is int8? What is the compute dtype? What are the various options (compute in int8, compute in other dtypes, etc). You can change this to be a separate section if you wish.